routing/Graph.py

```diff
@@ -0,0 +1,205 @@
+""" A Python Class
```

Comment:
Did you write this or find it off the internet? Need citation if copy pasta

Reply:
Citation?

Reply:
It's literally on line 6.

Reply:
Ah ok